src/components/TextInput/BaseTextInput.js

```diff
@@ -211,6 +213,11 @@ function BaseTextInput(props) {
         }
     };
 
+    const isNativeOS = () => {
```

Comment:
hey @samilabud - i mean to do like this src/libs/getPlaidOAuthReceivedRedirectURI